### PR TITLE
bug: able to add isolated event with invalid date for feb due to leap year 

### PR DESCRIPTION
**Pull request by gmengqi**
solution: check if it leap year and validate the date

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -191,8 +192,15 @@ public static LocalDate parseDate(String date) throws ParseException {
      * @throws ParseException to be thrown if there is error.
      */
     public static void checkDateExist(LocalDateTime date, String day) throws ParseException {
+        boolean isLeapYear = date.getYear() % 4 == 0;
```
Comment (clydelhui) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:195]: Might want to add a few extra conditions for this:
```
A year may be a leap year if it is evenly divisible by 4. 
Years that are divisible by 100 (century years such as 1900 or 2000) cannot be leap years unless they are also divisible by 400. 
(For this reason, the years 1700, 1800, and 1900 were not leap years, but the years 1600 and 2000 were.)
```
The above was taken from Google

**Review by clydelhui — CHANGES_REQUESTED** [review]
Mostly LGTM! Please do update the checks to make it accurate!

**Comment by lyndonlim27** [comment]
image

Could you also change the example help command for isolated event to a date very far into the future.

**Review by lyndonlim27 — COMMENTED** [review]
LGTM!

**Review by clydelhui — APPROVED** [review]

**Pull request merged**